src/windows/webview.c

```diff
@@ -477,21 +478,23 @@ static ICoreWebView2EnvironmentOptionsVtbl VTBL5 = {
 
 // Public
 
-static HMODULE webview_load_dll(void)
+static bool webview_dll_path(WCHAR *pathw, bool as_user)
 {
+	bool ok = false;
+
 	HKEY key = NULL;
-	WCHAR *dll = NULL;
-	HMODULE lib = NULL;
+	WCHAR dll[MTY_PATH_MAX] = {0};
+
+	// https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution#detect-if-a-webview2-runtime-is-already-installed
```

Comment:
It is good to provide source urls like this from a maintenance perspective. Thanks.